esx_service/vmdk_ops.py

```diff
@@ -153,12 +157,24 @@ def RunCommand(cmd):
 
     return (s, o)
 
+def is_valid_volume_name(name):
+    """ Check the given name is a valid volume name """
```

Comment:
Done. Remove this function and introduce a global variable "valid_volume_name_reg", use valid_volume_name_reg.match() to check directly.